datamodel/classes_02.tex

```diff
@@ -0,0 +1,78 @@
+\input{json}
+\begin{lstlisting}[caption={Klassen-Datenmodell Beispiel 2: Jahrgangstufe 11, Sekundarstufe 2},frame=tlrb]
+{
+ class: "KLASSE-11",
+ name: "Jarganstuffe 11",
+ school: "SCHULE-04",
+ school-year: "SJ-20/21",
+ start: "01-09-2020",
+ end: "31-08-2021",
+ grade: [ "11", ],
+ subjects: [  ],
+ students: [
```

Comment:
we use users: [{role:ROLE_ID,user:USER_ID},{...},...] to create dynamic roles for different group scenarios.

Reply:
No